### ISSUE TITLE
Do not list PDFs in the dataset list

### ISSUE THREAD

**Issue by takavarasha** [OCHA-DAP/hdx-ckan#2913]
Currently, users are encouraged to upload additional metadata for their data (if it exists) as an additional resource in their data set. This can sometimes lead to users uploading PDF files containing the metadata. The data set list will then contain pdf documents, which may give users the  incorrect impression that PDFs are ok. See image below.
![image](https://cloud.githubusercontent.com/assets/6885303/7915399/41e9cb10-084b-11e5-8bca-251d00890b18.png)

Consider filtering out PDF resources from the dataset list (or alternatively banning PDFs from resources and providing a separate mechanism for uploading additional metadata files)

**Comment by danmihaila** [comment]
@takavarasha @luiscape is this still a request we want to implement?

**Comment by takavarasha** [comment]
Yes, although I would consider this to be a low priority request.

**Comment by luiscape** [comment]
I would agree with @takavarasha's points above.
